scripts/setup-sudobot-pm2.sh

```diff
@@ -0,0 +1,264 @@
+#!/bin/bash
+
+echo "=== Setting up SudoBot without Docker using PM2 ==="
+
+# Reminder to have Discord credentials ready
+echo "Please ensure you have the following information from your Discord Developer Portal:"
+echo "- Bot Token"
+echo "- Client ID"
+echo "- Client Secret"
+echo "- Home Guild ID (the ID of your Discord server)"
+echo "- PostgresSQL Username/Password"
+echo "- Postgres Data"
+echo ""
+echo -n "Press Enter to continue once you have these credentials ready..."
+read  # Waits for the user to press Enter
+
+# Function to check if a command exists
+command_exists () {
+    command -v "$1" >/dev/null 2>&1 ;
+}
+
+# Function to install Node.js (v22.x)
+install_node() {
+    echo "Installing Node.js..."
+    curl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash -
+    sudo apt-get install -y nodejs
+}
+
+# Function to install Git
+install_git() {
+    echo "Installing Git..."
+    if command_exists apt-get ; then
+        sudo apt-get update
+        sudo apt-get install -y git
+    elif command_exists yum ; then
+        sudo yum install -y git
+    else
+        echo "Unsupported package manager. Please install Git manually."
+        exit 1
+    fi
+}
+
+# Function to install PostgreSQL
+install_postgresql() {
+    echo "Installing PostgreSQL..."
+    if command_exists apt-get ; then
+        sudo apt-get update
+        sudo apt-get install -y postgresql postgresql-contrib
+    elif command_exists yum ; then
+        sudo yum install -y postgresql postgresql-server postgresql-contrib
+        sudo postgresql-setup initdb
+        sudo systemctl enable postgresql
+        sudo systemctl start postgresql
+    else
+        echo "Unsupported package manager. Please install PostgreSQL manually."
+        exit 1
+    fi
+}
+
+# Function to install PM2
+install_pm2() {
```

Comment:
If you didn't know, SudoBot already comes with PM2 (take a look at the `package.json`) file. Instead of global install, running `npm install` or `bun install` in the project root installs `pm2` as well. Then you can run `npx pm2` or `bunx pm2` to execute PM2 commands.